.github/workflows/main.yml

```diff
@@ -43,33 +43,33 @@ jobs:
       run: |
         cd test
         npm test
-  coverage:
-    needs: [test-node]
-    runs-on: ubuntu-latest
-    timeout-minutes: 10
-    services:
-      mongodb:
-        image: mongo:4.2
-        ports:
-          - 27017:27017
-    strategy:
-      matrix:
-        node-version: [14.x]
-    steps:
-    - uses: actions/checkout@v2
-    - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v1
-      with:
-        node-version: ${{ matrix.node-version }}
-    - run: |
-        cd test
-        npm install
-    - name: Generate coverage report
-      run: |
-        cd test
-        npm run coverage-ci
-    - name: Upload coverage to Codecov
-      uses: codecov/codecov-action@v1
-      with:
-        file: ./test/coverage/lcov.info
-        fail_ci_if_error: true
+  # coverage:
+  #   needs: [test-node]
+  #   runs-on: ubuntu-latest
+  #   timeout-minutes: 10
+  #   services:
+  #     mongodb:
+  #       image: mongo:4.2
+  #       ports:
+  #         - 27017:27017
+  #   strategy:
+  #     matrix:
+  #       node-version: [14.x]
+  #   steps:
+  #   - uses: actions/checkout@v2
+  #   - name: Use Node.js ${{ matrix.node-version }}
+  #     uses: actions/setup-node@v1
+  #     with:
+  #       node-version: ${{ matrix.node-version }}
+  #   - run: |
+  #       cd test
+  #       npm install
+  #   - name: Generate coverage report
+  #     run: |
+  #       cd test
+  #       npm run coverage-ci
+  #   - name: Upload coverage to Codecov
+  #     uses: codecov/codecov-action@v1
+  #     with:
+  #       file: ./test/coverage/lcov.info
+  #       fail_ci_if_error: true
```

Comment:
@davidlehn -- isn't there a better solution to this CI stuff than just commenting the whole thing out?

Reply:
@dlongley  Is it okay if I open an issue for this and address this later and not on this PR?

Reply:
This is not a very exciting [failure](https://github.com/digitalbazaar/bedrock-web-wallet/runs/5235630929?check_suite_focus=true).  The problem is that there is only a bogus "should test something" test.  Which doesn't touch any of the real code, so the lcov.info output is an empty file.  And the tools fail to upload an empty file.

This could be "fixed" by just loading 'bedrock-web-wallet' in the test, to see ~zero coverage.  But it seems that needs some fixing.  A context package doesn't exist, peer deps are not installed in the test dir, and some code doesn't make it through webpack/babel.  Might need updated tools to fix that?  I'm not sure.

Reply:
Opened an issue here https://github.com/digitalbazaar/bedrock-web-wallet/issues/5